coordinator/src/node/expired_positions.rs

```diff
@@ -57,10 +58,7 @@ pub async fn close(node: Node) {
         };
 
         let closing_price = match BitmexClient::get_quote(&position.expiry_timestamp).await {
-            Ok(quote) => match position.direction {
-                trade::Direction::Long => quote.bid_price,
-                trade::Direction::Short => quote.ask_price,
-            },
+            Ok(quote) => closing_price(position.direction, quote),
```

Comment:
The `closing_price` is not wrong but for me it adds to the confusion in this whole story and I had to do some mental gymnastics to follow what is happening here 🙈 

What about we do something like this: 

```rust
// 1. get counter direction, i.e. he was long, now he needs to go short, or he was short, now needs to go long
let direction = position.direction.opposite();

// 2. TODO: get latest quote from the orderbook/counter party
let quote = BitmexClient::get_quote(&position.expiry_timestamp)
    .await
    .unwrap();

// 3. get closing price based on order direction
let closing_price = match direction {
    Direction::Long => quote.ask_price,
    Direction::Short => quote.bid_price,
};

```